can i get my pizza to people component into the mainline code

melsummy opened the pull request.

melsummy committed:
added anguler material

command was

ng add @angular/material

melsummy committed:
generate a components

command used

ng generate component pizza-calculator

melsummy committed:
updated the pizza calculator

to show when a another pizza should be order

melsummy committed:
added css

tsteelematc commented:
You have merge conflicts and have to merge the latest master into your PR branch and resolve... And you chose a different theme, so make sure to resolve appropriately and keep the theme of master...

Hope this makes sense, if not, let me know and I can help.

melsummy committed:
Merge branch 'master' into MFischer/week-07

melsummy commented:
@tsteelematc 
i do believe i have fixed the conflicts

tsteelematc commented:
Could you move your component to the bottom...

And not make changes to the original Pizza Toppings component in master...

Hope that makes sense. Let me know if you have questions...

<img width="1026" alt="image" src="https://github.com/user-attachments/assets/78fcbdf6-9424-47a0-b8df-d352cd038be9">

melsummy committed:
changed back

melsummy committed:
i think i fixed it

melsummy commented:
@tsteelematc 
I think i fixed the problem
I did have some trouble as I did something, that I'm not really sure how but I think I fixed it. 
otherwise please let me know

tsteelematc commented on src/app/pizza-toppings/pizza-toppings.component.html:
```diff
@@ -33,6 +33,9 @@
+</mat-card>
+
+
+<app-pizza-calculator></app-pizza-calculator>
```
I think you want to put this in app.component.html, not in the pizza-toppings components

melsummy replied:
@tsteelematc 
done

melsummy committed:
moved to the app.component.html

melsummy committed:
Merge branch 'master' into MFischer/week-07

melsummy commented:
@tsteelematc 
ok i think i get it now

melsummy committed:
removed the CSS

comment out the CSS from the pizza-calculator for better merging

melsummy committed:
Merge branch 'MFischer/week-07' of https://github.com/2024-fall-jsf-online/components into MFischer/week-07

melsummy commented:
@tsteelematc 
comments out the CSS

melsummy committed:
work pls
